libs/velo-external-db-core/src/utils/schema_utils.ts

```diff
@@ -217,14 +217,20 @@ export const fieldKeyToPrecision = (fieldKey: string): number | undefined => {
     return FieldsWithPrecision.includes(fieldKey) ? 255 : undefined
 }
 
-export const wixFormatFieldToInputFields = (field: collectionSpi.Field): InputField => ({
+export const wixFormatFieldToInputFields = (field: collectionSpi.Field): InputField => ( field.encrypted ? encryptedInputField(field.key) : 
+{
```

Comment:
It took me a few moments to understand what's going on here, is it possible to extract the creation of a non-encrypted field into a function like you did for an encrypted column?

Reply:
[nonEncryptedInputField](https://github.com/wix/velo-external-db/pull/509/commits/a9ab1c7d3bdf2e08bf9e35021ba0025cc4becc6c)